src/servers/sapphire_zone/Zone/InstanceContent.cpp

```diff
@@ -222,6 +222,8 @@ void Core::InstanceContent::registerInstanceObj( Core::Entity::InstanceObjectPtr
    object->setParentInstance( InstanceContentPtr( this ) );
 
    m_instanceObjects[object->getId()] = object;
+
+   g_log.debug("Registered instance eobj: " + std::to_string( object->getId() ) );
```

Comment:
ÿ

Reply:
i didn't know if it worked or not and attaching a debugger is too big brained for me :(